benchmark/benchmark_revisions.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Benchmarking tool for different commits
+"""
+# pylint: disable=subprocess-run-check
+import argparse
+import os
+import subprocess
+
+# ANSI escape sequences for terminal colors
+Colors = {
+    "red": "\033[31m",
+    "yellow": "\033[33m",
+    "blue": "\033[34m",
+    "magenta": "\033[95m",
+}
+RESET = "\033[0m"
+
+
+def col(text, color):
+    """Wraps the given text in color ANSI sequences.
+
+    Args:
+        text  (str): text to print
+        color (str): ANSI color code
+
+    Returns:
+        str: text wrapped with ANSI codes
+    """
+    return Colors[color] + text + RESET
```

Comment:
Adding `from benchmark import col, Colors, RESET` seemed to have worked fine for me
```suggestion
```

Reply:
Perfect!